source/Octopus.Manager.Tentacle/Octopus.Manager.Tentacle.csproj

```diff
@@ -4,25 +4,34 @@
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <RootNamespace>Octopus.Manager.Tentacle</RootNamespace>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <IsPublishable>false</IsPublishable>
+    <IsPublishable>true</IsPublishable>
     <OutputPath>bin</OutputPath>
     <OutputType>WinExe</OutputType>
     <PublishDir>../../_build/$(AssemblyName)/$(TargetFramework)/$(RuntimeIdentifier)</PublishDir>
-    <RuntimeIdentifiers>win</RuntimeIdentifiers>
-    <TargetFramework>net48</TargetFramework>
     <UseWindowsForms>true</UseWindowsForms>
     <UseWPF>true</UseWPF>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <EnableWindowsTargeting>true</EnableWindowsTargeting>   
+    <TargetFrameworks>net48;net6.0-windows</TargetFrameworks>
+    <SelfContained>false</SelfContained>
```

Comment:
What happens if `SelfContained == true`? Using false here might work fine as Tentacle .NET 6 is self-contained, but maybe we would be communicating our intent better by setting this to `true`.

Reply:
`Self-Contained` was never specified during Tentacle built previously. So it was defaulted to `false`.
When we target `net6.0-windows`, `Self-Contained` is set to `true` by default. This leads to an error saying that self-contained build can't have a dependency on non-self-contained project (Tentacle). 

If we'd like to set this to `true` for Tentacle Manager, Octopus.Tentacle will also need to be self-contained. Are we sure we want this?

Reply:
Changing `Octopus.Tentacle` to be self-contained will lead to:
- Its dependencies must also be self-contained. eg. `Octopus.Tentacle.Contracts`
- An increase in tentacle package size.

Reply:
I was under the impression that Tentacle on .NET 6 was already self-contained, as that was a requirement for Philips when the work was initially done.

I might be getting my wires crossed here. Let's discuss further on slack.